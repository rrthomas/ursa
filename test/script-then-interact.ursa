for i of my_range(5) { print(i) }