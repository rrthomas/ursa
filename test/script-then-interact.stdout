0
1
2
3
4
0
1
2
3
4
null
null
0
1
5
5
> 0
1
2
3
4
null
> 